packages/openapi-ts/src/types/config.d.ts

```diff
@@ -135,6 +135,23 @@ export interface UserConfig {
     | (string & {})
     | Record<string, unknown>
     | Input;
+  /**
```

Comment:
What's up with this diff?

Reply:
I have no idea. I merely ran the linters / formatters at the pre commit

Reply:
Reverting this accidental diff is the last thing!